hr_time/public/scss/hr_time.bundle.scss

```diff
@@ -1,2 +1,32 @@
+@import 'mixins';
 @import "./dashboard";
-@import "./checkin_status";
\ No newline at end of file
+@import "./checkin_status";
+
+.modal .modal-body{
+    padding-bottom: 0 !important;
+}
+
+label#worklog_section_label{
+    margin-bottom: 0 !important;
+}
+
+.worklog-status-alert{
+    @include flex-center;
+    @include badge;
+    opacity: 0.9;
+    font-size: 0.7rem;
```

Comment:
```suggestion
    font-size: var(--text-md);
```

Reply:
I have used var(--text-sm) instead because the alert message in the mockup you shared, looks like it is using smaller font size than medium.
Can be changed upon review.